data/cameras.xml

```diff
@@ -11632,7 +11632,14 @@
 		<Sensor black="142" white="4095"/>
 		<Aliases>
```

Comment:
Looks like you need to touch `data/cameras.xsd`, line 310, bump 5 to 9.

Reply:
Ta, done.